src/player.cpp

```diff
@@ -682,7 +760,7 @@ ActiveSound *Player::findByHandle(SoLoud::handle handle)
         int index = 0;
         while (index < (int)sounds[i].get()->handle.size())
         {
-            if (sounds[i].get()->handle[index] == handle)
+            if (sounds[i].get()->handle[index].handle == handle)
```

Comment:
Oooh, a bug fix?

Reply:
nono! That's the new struct `ActiveHandle` which contains the `handle` and the `bufferingTime` :)
Used in `ActiveSound` as a list of active handles.